test/e2e_node/e2e_service.go

```diff
@@ -252,6 +252,8 @@ func (es *e2eService) startKubeletServer() (*killCmd, error) {
 		cmdArgs = append(cmdArgs,
 			"--runtime-cgroups=/docker-daemon",
 			"--kubelet-cgroups=/kubelet",
+			"--cgroup-root=/",
```

Comment:
on systemd node cgroup_root should set to system.slice, not "/". The change move the initial the block of code "--cgroup-root="/"" just simply breaks container vm image, but not fixing systemd nodes at all.